guides/micronaut-microservices-services-discover-eureka/bookcatalogue/groovy/src/main/groovy/example/micronaut/BooksController.groovy

```diff
@@ -8,12 +8,11 @@ import io.micronaut.http.annotation.Get
 @Controller("/books") // <1>
 class BooksController {
 
-    @Get("/") // <2>
+    @Get // <2>
     List<Book> index() {
-        [
-            new Book("1491950358", "Building Microservices"),
-            new Book("1680502395", "Release It!"),
-            new Book("0321601912", "Continuous Delivery:"),
-        ]
+        Book buildingMicroservices = new Book("1491950358", "Building Microservices")
```

Comment:
Fine with this change. However, I think the other syntax without variables was nicer though.